org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServers.java

```diff
@@ -300,7 +300,6 @@ public long getStartVersion() {
 			return this.startVersion;
 		}
 
-
```

Comment:
can we remove LanguageServers from the PR? By now it is only a line which has been removed :)